b2sdk/file_lock.py

```diff
@@ -249,7 +252,7 @@ def add_to_upload_headers(self, headers):
 
 class BucketRetentionSetting:
     """Represent bucket's default file retention settings, i.e. whether the files should be retained, in which mode
```

Comment:
I thought the right way to format this would be 
```
"""
Represent bucket's default file retention settings, i.e. whether the files should be retained, in which mode...
"""
```

because both the original and the changed one look awful.